### ISSUE TITLE
[TTT2] Attempt to index a nil value

### ISSUE THREAD

**Issue by Delta7x** [yupi2/passive_footvision#1]
Suppose I'm not really surprised since it's almost 10 years old but, unfortunately, after installing this to my server I get the following error:

[passive_footvision-master] addons/passive_footvision-master/lua/cl_passive_footvision.lua:18: attempt to index a nil value
    1. fn - addons/passive_footvision-master/lua/cl_passive_footvision.lua:18
        2. Run - addons/ulib/lua/ulib/shared/hook.lua:109
            3. CheckForLegacyTabs - addons/ttt2-0.14.0b/lua/terrortown/menus/gamemode/legacy.lua:86
                4. Initialize - addons/ttt2-0.14.0b/lua/terrortown/menus/gamemode/legacy.lua:105
                    5. PostInherit - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/client/cl_help.lua:105
                        6. BuildFromFolder - addons/ttt2-0.14.0b/lua/ttt2/libraries/classbuilder.lua:123
                            7. unknown - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/client/cl_help.lua:108
                                8. include - [C]:-1
                                    9. ttt_include - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/shared/sh_include.lua:218
                                        10. unknown - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/client/cl_main.lua:125
                                            11. include - [C]:-1
                                                12. ttt_include - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/shared/sh_include.lua:218
                                                    13. unknown - addons/ttt2-0.14.0b/gamemodes/terrortown/gamemode/cl_init.lua:3

**Comment by yupi2** [comment]
Yeah, it predates any TTT alternative (that was worth using at least) by a bit, so the hardcodey settings-tab option-adding doesn't work with things like TTT2 (which is a shame since that *does* call the hook).
You can comment-out the whole hook or delete it and then the error won't pop up lol but I don't know if the rest of the code will still work.

It's not something that I'll try to fix but I'll leave the issue open in case anyone else stumbles upon it.
